Makefile

```diff
@@ -43,6 +43,7 @@ lint: ## Run golangci-lint against codes.
 	$(GOLANGCI_LINT) run
 
 test: fmt vet ## Run tests.
+	go mod tidy
```

Comment:
I suggest we do not place command `go mod tidy` here.

Reply:
Why not?

Reply:
I think it's a dependency issue and not affect test process.

Reply:
Everyone should to check the following list:

* code format
* go mod
* vet
* others

So, the test is not just about the unit test. And `fmt` and `vet` are also not be part of the test process.